build.gradle.kts

```diff
@@ -101,15 +101,32 @@ java {
 }
 
 stutter {
-    isSparse = (findProperty("stutter.sparce")?.toString()?.toBoolean()) ?: true
-    java(8) {
-        compatibleRange("5.0")
-    }
-    java(11) {
-        compatibleRange("5.0")
-    }
-    java(17) {
-        compatibleRange("7.3")
+    sparse.set(providers.gradleProperty("stutter.sparce").map(String::toBoolean).orElse(true))
+    matrices {
+        register("java8") {
+            javaToolchain {
+                languageVersion.set(JavaLanguageVersion.of(8))
+            }
+            gradleVersions {
+                compatibleRange("5.0")
```

Comment:
5.0 is no longer supported after #222 is merged.

Reply:
Rebased, fixed versions, and regenerated locks again to match master.